apps/files/src/components/FilesTopBar.vue

```diff
@@ -104,6 +104,9 @@ export default {
       let path = this.item === '' ? '/' : `${this.item}/`
       return this.$client.files.getFileUrl(`/${path}`)
     },
+    newFolderErrorMessage () {
+      return this.checkNewFolderName(this.newFolderName)
```

Comment:
So this function is called twice. Looks strange.

Reply:
I know what you mean... I thought I should use a computed variable for binding and a method if the logic part gets more complex later on. Computed properties should stay simple

Reply:
We could also use event @keydown. That way we could get rid for example of the watch and take care of this in the method. Also, we could implement this way a little bit simpler that if the user tries to write an invalid character, it's not written thanks to preventDefault() and he just gets the error that it was invalid. That way he wouldn't have to delete it afterwards. But I guess this is opened for discussion if it would be the right approach in regards to UX.